app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -135,9 +135,12 @@ def analyze
 
   def collect_compliance_data(image)
     return unless image_inspector_client.respond_to? :fetch_oscap_arf
+    _log.info "collecting compliance data for #{options[:docker_image_id]}"
     openscap_result = image.openscap_result || OpenscapResult.new(:container_image => image)
     openscap_result.attach_raw_result(image_inspector_client.fetch_oscap_arf)
     openscap_result.save
+  rescue ImageInspectorClient::InspectorClientException => e
```

Comment:
Are we requiring [image-inspector-client](https://github.com/moolitayer/image-inspector-client/blob/65e4552fb61d5f347ce052d40cdafe61a3b6f05a/lib/image-inspector-client.rb#L79) by this time?  I'd assume image_inspector_client would not be set if this constant was not already defined, but thought I would ask anyway.